src/kibit/driver.clj

```diff
@@ -84,3 +84,62 @@
   (if (zero? (count (apply run source-paths rules args)))
     (System/exit 0)
     (System/exit 1)))
+
+(defn exec
+  "Given a [Clojure CLI-style map][Execute a function] `options`, turn this map
+  into an equivalent set of options as expected by run and external-run above.
+
+  Please note that rules is not supported. nil is passed to external-run below
+  which enables all rules (see kibit.rules/all-rules). I don't know what rules
+  should be. A list of rule names as strings doesn't seem to be it.
```

Comment:
I'd be happy to support rules in this pull-request. Could someone tell me what this should look like on the command-line?

Reply:
Kibit currently doesn't support rule names.